config/config.go

```diff
@@ -996,3 +1003,10 @@ func (c *Config) parseHistogramBuckets() ([]float64, error) {
 	sort.Float64s(result)
 	return result, nil
 }
+
+func (c *Config) parseEnv() {
+	// Set Redis password from environment variable if not set earlier (configuration file)
+	if c.SwarmRedisPassword == "" {
+		c.SwarmRedisPassword = os.Getenv("SWARM_REDIS_PASSWORD")
```

Comment:
`redisPasswordEnv`

Reply:
This is what happens when you don't pay attention and think you can multitask :sweat_smile: